docs/src/rust-feature-support.md

```diff
@@ -200,3 +200,12 @@ related to [advanced features](#advanced-features).
 
 Please refer to [Intrinsics](rust-feature-support/intrinsics.md) for information
 on the current support in Kani for Rust compiler intrinsics.
+
+### Floating point operations
+
+Kani supports floating point numbers, but some kinds of operations on floats are "over-abstracted."
+This means the verifier can raise errors that cannot actually happen when the code is run normally.
+For instance, ([#1342](https://github.com/model-checking/kani/issues/1342)) the sin/cos functions basically return a nondeterministic value between -1 and 1.
+In other words, they largely ignore their input and give very conservative answers.
+This range certain includes the "real" value, but it means Kani could raise spurious errors that cannot actually happen.
+This makes Kani unsuitable for verifying some kinds of properties (e.g. precision) about numerical algorithms.
```

Comment:
Should we include a suggestion of using stubs to replace the usage of those functions with versions that suit the verification needs?

Reply:
I initially decided no, but I thought again and went for it. :)